README.md

```diff
@@ -1 +1,51 @@
 # INF 6420 - Intro to Web Development
+ Rylie Butner's INF6420 Project
+
+ A portfolio site showcasing my work.
+
+ ## Wireframes
+
+ These wireframes show how I plan to structure my site.
+
+ ![Wireframe of Landing Page](wireframes\landing-page.jpg)
```

Comment:
Hi @ryliebutnerWSU use a `/` instead of a `\` to fix the broken images.

Reply:
@axamei my bad! I copied the file path and forgot to change that. weirdly the images previewed fine in VS code.

Reply:
@axamei (links are fixed)